src/server/game/Entities/Player/Player.cpp

```diff
@@ -15852,6 +15858,17 @@ void Player::RemoveRewardedQuest(uint32 questId, bool update /*= true*/)
         m_RewardedQuestsSave[questId] = QUEST_FORCE_DELETE_SAVE_TYPE;
     }
 
+    // Remove seasonal quest also
+    if (Quest const* qInfo = sObjectMgr->GetQuestTemplate(questId))
+    {
+        uint16 eventId = sGameEventMgr->GetEventIdForQuest(qInfo);
```

Comment:
https://travis-ci.org/TrinityCore/TrinityCore/builds/197388841
```
src/server/game/Entities/Player/Player.cpp:15864:41: fatal error: 
      no member named 'GetEventIdForQuest' in 'GameEventMgr'
        uint16 eventId = sGameEventMgr->GetEventIdForQuest(qInfo);
                         ~~~~~~~~~~~~~  ^
```
I would have thought that it would be enough that `GetEventIdForQuest` is defined in `GameEventMgr.h` (which is in the #include list in this file). I presume you know why this one fails. Perhaps an additional declaration is needed?